src/argo.h

```diff
@@ -15,8 +15,9 @@
 /**
  * @brief initialize ArgoDSM system
  * @param argo_size the desired size of the global memory in bytes
+ * @param cache_size the desired size of the local cache in bytes
```

Comment:
A better C interface would be desired. Do we have to fix all of that in this pull-request or should be postpone that to a separate "fix the C interface" patch?

Reply:
No, I don't think it is necessary to bring the documentation of the C interface entirely up to speed with the C++ one right now. However, since the special meaning of the value `0` was introduced in this patch it would be nice to have some mention of it here as well. Just put something like "a value of 0 will set <parameter> according to <environment variable>".

Reply:
fixed in f08ae31253869967074ad6629d77440989b6aafa